app/src/main/java/com/example/parkingsystemjava/mvp/presenter/ParkingReservationPresenter.java

```diff
@@ -80,4 +80,9 @@ public void validateAndSaveReservation(String entryDate, String exitDate, String
             view.closeScreen();
         }
     }
+
+    @Override
+    public void deleteOldReservations() {
+        model.deleteOldReservations();
```

Comment:
you can add here a toast on the view to let know the user that something happened

Reply:
done